src/app/3d/qgs3dmapcanvaswidget.h

```diff
@@ -90,6 +100,7 @@ class APP_EXPORT Qgs3DMapCanvasDockWidget : public QgsDockWidget
     QAction *mActionEnableShadows = nullptr;
     QAction *mActionEnableEyeDome = nullptr;
     QToolButton *mBtnOptions = nullptr;
+    QgsDockableWidgetHelper *mDockableWidget = nullptr;
```

Comment:
Sorry I wasn't clear, I meant to also rename this to `mDockableWidgetHelper` so it is clear it's not a QWidget, but rather some other class